language/German/strings.po

```diff
@@ -6463,6 +6501,8 @@ msgid ""
 "Enter combination or press %s to cancel\n"
 "%s"
 msgstr ""
+"Kombination eingeben oder %s zum Abbrechen drücken\n"
+"%s"
 
 #, c-format
 #~ msgid "System: \"%1$s\" software list "
```

Comment:
You can nuke the rest of the file from this point onwards – the obsolete translations will still be in version control history.

Reply:
I was going to ask about this, thanks. Fixed.